ansible/roles/postgresql/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+# PostgreSQL-related settings
+postgresql_is_local: "{{ 'true' if project_db_host | search( 'localhost|127\\.0\\.0\\.1|^/') else 'false' }}"
```

Comment:
Also can you talk about why you're using strings instead of Python's booleans `True` and `False`?